docker/entrypoint.sh

```diff
@@ -44,6 +44,14 @@ case "$1" in
   	exec /usr/bin/weed $@ $ARGS
 	;;
 
+  'replication-job')
+	MASTER=${WEED_MASTER-localhost:9333}
```

Comment:
Is this correct? Not sure.

master:9333-localhost:9333

Reply:
That just sets the default master to `localhost:9333` so that it is the same as the default for the `weed shell` command. You can override the setting by setting the WEED_MASTER environment variable, like in the compose file.